src/compiler/diagnosticMessages.json

```diff
@@ -3264,6 +3264,10 @@
         "category": "Error",
         "code": 2802
     },
+    "Property '{0}' does not exist on type '{1}'. Try changing the `lib` compiler option to include 'dom'.": {

```

Comment:
Use consistent quoting style around non-translated portions.

Reply:
@DanielRosenwasser I'm going off of existing precedent here for backticks around `lib` and quotes around others: codes 2550, 2583, 2584, and 2585. I'm not clear what the requested change is, sorry?

Reply:
I think changing all of lib's backticks to single quotes is better. The oldest backticks I could find are all less than a year old.

Actually, the only backticks that make sense to me are the ones around long commands like `npm i -D @types/{1}` or long code snippets like `declare module '{0}'` (which is probably incorrectly double-quoted). There's even a message with mismatched backticks/single quotes. We should make a cleanup sweep to restore single quotes.